File: Loop_Day/task_one.py

total = 0
sum_odd = 0
sum_even = 0
square_odd = 0
square_even = 0
mean = 0
sum_square_even = 0
sum_square_odd = 0
sum_square = 0
   
for number in range(10):
    value = int(input('Enter a number: '))
  

    if value % 2 == 0:
        sum_even += value
        square_even *= value
        print("It is an even number") 
    else :
        sum_odd += value
        square_odd *= value
        print(" It is an odd number")
    total = sum_even + sum_odd
    mean = total / 10
    sum_square_odd += square_odd
    sum_square_even += square_even
print(f'sum of even number is {sum_even}')
print(f'sum of odd number is {sum_odd}')

File: Loop_Day/TaskOne.java

import java.util.Scanner;
public class TaskOne{
 public static void main(String[] args){
   
    Scanner input = new Scanner(System.in);
    int sumEven = 0;
    int sumOdd = 0;
    int total = 0;
    int squareOdd = 0;
    int squareEven = 0;
    double mean = 0;
    int sumSquareEven = 0;
    int sumSquareOdd = 0;
    int sumSquare = 0;
   
    

    for(int count = 0; count < 10; count++){
    System.out.println("Enter a number: ");
    
    int number =  input.nextInt();
   
    if (number % 2 == 0){
      
        sumEven += number;
        squareEven = number * number;
        sumSquareEven += squareEven;
        System.out.printf("it is an even number are %d%n",number);
         
        System.out.printf (" The square of  Even number is %d%n",squareEven); 
    
   }  
    else {
       
        sumOdd += number;
         squareOdd = number * number;
          sumSquareOdd += squareOdd; 
         System.out.printf("it is an odd number are %d%n",number); 
         System.out.printf(" The square of each odd number is %d%n",squareOdd); 

}


}

    total = sumOdd + sumEven;
    mean = total / 10;
    sumSquare = sumSquareEven + sumSquareOdd;



    
   
    System.out.printf(" sum of even number is %d%n",sumEven);  
    System.out.printf(" sum of odd number is %d%n",sumOdd); 
    System.out.printf(" sum of all number is %d%n",total); 
   
  
    System.out.printf (" The mean of the numbers is %.2f%n",mean); 
    System.out.printf (" The sum of the square of even number is %d%n",sumSquareEven); 
    System.out.printf (" The sum of the square of odd number is %d%n",sumSquareOdd); 
    System.out.printf (" The sum of the square of all is %d%n",sumSquare); 


}

}








